rbx_binary/src/deserializer.rs

```diff
@@ -1127,7 +1127,7 @@ impl<R: Read> BinaryDeserializer<R> {
 
                     // Roblox writes a type marker for CFrame here that we don't
                     // need to use.
-                    chunk.read_u8()?;
+                    assert!(chunk.read_u8()? == Type::CFrame as u8);
```

Comment:
Oh, sorry, I meant to return an error here! We don't want to panic un malformed input if we can.